src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,88 @@
-export const Pagination = () => {};
+import * as React from 'react';
+import { getNumbers } from '../../utils';
+
+type Props = {
+  items: string[];
+  perPageItems: string;
+  selectedPage: number;
+  maxPage: number;
+  handlerSelectedPage: (numb: number) => void;
+};
+
+export const Pagination = ({
+  items,
+  perPageItems,
+  selectedPage,
+  maxPage,
+  handlerSelectedPage,
+}: Props) => {
+  return (
+    <>
+      <ul className="pagination">
+        <li className={`page-item ${selectedPage === 1 ? 'disabled' : ''}`}>
+          <a
+            data-cy="prevLink"
+            className="page-link"
+            href="#prev"
+            aria-disabled={selectedPage === 1}
+            onClick={() => {
+              if (selectedPage !== 1) {
+                handlerSelectedPage(selectedPage - 1);
+              }
+            }}
+          >
+            «
+          </a>
+        </li>
+        {getNumbers(1, maxPage).map(n => {
+          return (
+            <li
+              className={`page-item ${n === selectedPage ? 'active' : ''}`}
```

Comment:
why you are using ternary operator? is here possible to use another styles? just make it like this `className: condition`